code_survey/controllers/main.py

```diff
@@ -53,10 +53,8 @@ def survey_submit(self, survey_token, answer_token, **post):
             answer, comment = self._extract_comment_from_answers(question, post.get(str(question.id)))
             errors.update(question.validate_question(answer, comment))
             if not errors.get(question.id):
-
                 if('is_test' in post):
```

Comment:
```suggestion
                if 'is_test' in post:
```
According to guidelines, I think if an if statement just makes one check it should not have parenthesis :+1:

Reply:
Same as before, this comes from standard